### PR TITLE
[1LP][RFR] Add providers and rename tests

### PR DESCRIPTION
**Pull request by t-fischer**
Renamed all tests to have them starting with sdn_*
Fix a button. 
Add more provider coverage
{{pytest: -v -q cfme/tests/networks/test_sdn_balancers.py cfme/tests/networks/test_sdn_crud.py cfme/tests/networks/test_sdn_downloads.py cfme/tests/networks/test_sdn_navigation.py cfme/tests/networks/test_sdn_ports.py --use-provider=complete}}

### CODE REVIEW

File: cfme/tests/networks/test_sdn_balancers.py [hunk 1]
```diff
@@ -41,7 +42,8 @@ def test_prov_balances_number(network_prov_with_load_balancers):
         assert int(balancers_number) == sum_all
 
 
-def test_balances_detail(provider, network_prov_with_load_balancers):
+@pytest.mark.meta(blockers=[BZ(1544411)], unblock=lambda provider: provider.type != 'gce')
```
Comment (quarckster) [inline cfme/tests/networks/test_sdn_balancers.py]: **REQUIRED**
```python
@pytest.mark.meta(blockers=[BZ(1544411)], unblock=lambda provider: provider.one_of(GCEProvider))
```

File: cfme/tests/networks/test_sdn_balancers.py [hunk 2]
```diff
@@ -2,21 +2,22 @@
...
+    pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider, GCEProvider],
                          scope='module')
 ]
 
 
 @pytest.fixture(scope='module')
-def network_prov_with_load_balancers(appliance):
-    prov_collection = appliance.collections.network_providers
+def network_prov_with_load_balancers(provider):
+    prov_collection = provider.appliance.collections.network_providers
```
Comment (quarckster) [inline cfme/tests/networks/test_sdn_balancers.py]: **QUESTION**
It's not clear why did you change it?

**Comment by quarckster** [comment]
Please do not forget to rename test cases in Polarion after merging this PR.

File: cfme/tests/networks/test_sdn_balancers.py (hunk 2 above)
Comment (t-fischer) [reply cfme/tests/networks/test_sdn_balancers.py]: Because with appliance, the fixture was setup only once and never changing, instead of for each provider

File: cfme/networks/views.py [hunk 3]
```diff
@@ -345,7 +345,7 @@ class NetworkRouterDetailsToolBar(View):
     """ Represents provider toolbar and its controls """
     configuration = Dropdown(text='Configuration')
     policy = Dropdown(text='Policy')
-    download = Button(title='Download')
+    download = Button(title='Download summary in PDF format')
```
[the remark this answers is not in the document]
Comment (t-fischer) [reply cfme/networks/views.py]: Yes.
test was also successful on 5.8
